typewise_alert.py

```diff
@@ -43,4 +57,5 @@ def send_to_email(breachType):
 def check_and_alert(alertTarget, coolingType, temperatureInC):
   breachType =\
     classify_temperature_breach(coolingType, temperatureInC)
-  return receiver[alertTarget](breachType)
+  receiver[alertTarget](breachType)
```

Comment:
Even if you remove the `receiver[alertTarget]` call, your tests will still pass.
So even when your tests pass, you will need an integrated environment to prove your code.
Strengthen the tests to prove your code without having to integrate it with 3rd party emailers, controllers, etc.